policy/modules/system/systemd.te

```diff
@@ -538,6 +550,8 @@ dev_write_sysfs_dirs(systemd_generator_t)
 dev_read_urand(systemd_generator_t)
 dev_create_sysfs_files(systemd_generator_t)
 dev_write_sysfs(systemd_generator_t)
+# for systemd-ssh-generator
+dev_read_generic_chr_files(systemd_generator_t)
```

Comment:
Which devices are these? It would be best to fix the labeling.

Reply:
It is /dev/vsock. But I don't see any specific label for this device.

Reply:
I have seen /dev/vsock on VMs running in vSphere.  Specifically I was using RHEL9 and had the 'open-vm-tools' rpm installed (not sure that was important).  I had started looking into update the refpolicy 'vmware' module to support, but kind of stopped a while ago (never made a pull request). 

For /dev/vsock it looks like the label should be vmware_device_t (in devices.fc)

Reply:
It seems that vsock can also be used for other virtual machines (qemu&kvm/HyperV) besides vmware:  Maybe the vsock_device_t label is more general.

Reply:
`vsock_device_t` works for me.